src/main/java/org/assertj/core/internal/Strings.java

```diff
@@ -330,6 +330,44 @@ private boolean areEqualIgnoringCase(CharSequence actual, CharSequence expected)
   }
 
   /**
+   * Verifies that two {@code CharSequence}s are equal, ignoring case considerations.
+   *
+   * @param info contains information about the assertion.
+   * @param actual the actual {@code CharSequence}.
+   * @param expected the expected {@code CharSequence}.
+   * @throws AssertionError if the given {@code CharSequence}s are not equal.
+   */
+  public void assertEqualsIgnoringWhitespaces(AssertionInfo info, CharSequence actual, CharSequence expected) {
+      if (areEqualIgnoringWhitespaces(actual, expected)) {
```

Comment:
I have oriented myself on the method assertEqualsIgnoringCase. Should this method also be simplified? Otherwise it feels i little bit inconsistent.